docs/specs/forecast-api.md

```diff
@@ -1,8 +1,8 @@
 | **Description**  | (EXTERNAL) Forecast API v1 |
-|:--|:--|
-| **Date** | Jul 2024 |
-| **Owner(s)** | Nick Masson |
-| **Status**  | Complete |
+|:--|:---------------------------|
+| **Date** | October 2024               |
```

Comment:
Should we add a revision table to the other `forecast-data` spec?

Or should we remove this one? 

If we decide to keep this one (and add one to the other doc), can we move below the top heading (# Contrail Forecast API) so it shows up better in the pdf rendering?

Reply:
I think it makes sense to remove it here. I usually like these for static docs ("released"), but these feel more like living docs... so seems less appropriate.

I'll strike it from the md before merging